test/classes/delete-free/owned-init2.good

```diff
@@ -1 +1,2 @@
-{x = 1} {x = 2} {x = 3}
+owned-init2.chpl:14: In function 'main':
+owned-init2.chpl:17: error: illegal lvalue in assignment
```

Comment:
It looks like this changes the .bad output to be the expected output for the test.  Have we verified that this was an oversight?